utils/common/src/test/java/org/apache/brooklyn/util/time/TimeTest.java

```diff
@@ -152,7 +152,13 @@ public void testElapsedSince() {
         Assert.assertFalse(Time.hasElapsedSince(aFewSecondsAgo, Duration.TEN_SECONDS));
         Assert.assertTrue(Time.hasElapsedSince(-1, Duration.TEN_SECONDS));
     }
-    
+
+    @Test
+    public void testParseTime() {
+        Assert.assertEquals(Time.parseElapsedTime("1s"), 1000);
```

Comment:
There's a lot of logic in `parseElapsedTimeAsDouble`. Would be good to test more scenarios, e.g. the inverse of of the tests in `testMakeStringExact_secondsAndMillis()`.